src/service/resource.js

```diff
@@ -358,7 +358,7 @@ export class Resource {
    * @param {!../layout-rect.LayoutMarginsChangeDef=} opt_newMargins
    */
   changeSize(newHeight, newWidth, opt_newMargins) {
-    this.element./*OK*/ changeSize(newHeight, newWidth, opt_newMargins);
+    this.element./*OK*/ applySize(newHeight, newWidth, opt_newMargins);
```

Comment:
Can /*OK*/ be removed?

Reply:
What does `/*OK*/` do? I really tried to Google this but I had a ton of trouble getting meaningful results.

Reply:
It breaks the regex in our presubmit checks

Reply:
I left the presubmit check for restricting `applySize`  because extension owners should use the `Mutator` APIs. There are a couple stragglers that need to be fixed in amp-list I think.